### DISCUSSION TITLE
HTTP Error 500.0 - ANCM In-Process Handler Load Failure

### DISCUSSION THREAD

**Discussion by CliffordShivkar** [dotnet/aspnetcore#26139] [General]
Hello Everyone!

I get the above error and the Hello World page doesn't load. This is a basic Hello World application.
I have VS 2017 ver 15.9.27 installed
 .Net Core SDK 2.2.110 installed and in the application, I installed Microsoft.AspNetCore.App 2.2.8 
I later installed Microsoft.AspNetCore.All 2.2.8, same Error. 

Please help. Thanks & Best Regards,
Clifford

**Comment by Tratcher** [comment]
2.2 is out of support, please try again with 3.1.

**Reply by CliffordShivkar** [reply]
Hi Tratcher, 
Thanks for replying.  Will 3.1 work with Visual Studio 2017 ver 15.9.27?

**Reply by Tratcher** [reply]
> Visual Studio 2019 16.4 was also released today and includes .NET Core 3.1. It is a required update to use .NET Core 3.1 with Visual Studio

https://devblogs.microsoft.com/dotnet/announcing-net-core-3-1/
